Sources/AST/Environment/Environment+Type.swift

```diff
@@ -52,6 +52,18 @@ extension Environment {
     }
   }
 
+  public func type(of externalCall: ExternalCall,
```

Comment:
Considering how you only use this on line 243 and do some nil coalescing there, is there any need for this function?

Reply:
Done :)